### ISSUE TITLE
windows-forms-accessibility-improvements should be moved to .NET 5

### ISSUE THREAD

**Issue by adegeo** [dotnet/docs-desktop#140]
This article should move to the .NET Core/5 area.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a62069ab-d5ff-3954-c834-f8c2fb41a70b
* Version Independent ID: 29fb3fc4-ec87-9317-e43d-fbeece4083d8
* Content: [Windows Forms Accessibility Improvements - Windows Forms .NET Framework](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/windows-forms-accessibility-improvements?view=netframeworkdesktop-4.8)
* Content Source: [dotnet-desktop-guide/framework/winforms/windows-forms-accessibility-improvements.md](https://github.com/dotnet/docs-desktop/blob/master/dotnet-desktop-guide/framework/winforms/windows-forms-accessibility-improvements.md)
* Product: **dotnet-framework**
* Technology: **dotnet-winforms**
* GitHub Login: @M-Lipin
* Microsoft Alias: **adegeo**
